hazelcast/src/main/java/com/hazelcast/config/CardinalityEstimatorConfig.java

```diff
@@ -51,6 +58,8 @@
 
     private String quorumName;
 
+    private MergePolicyConfig mergePolicyConfig = DEFAULT_MERGE_POLICY_CONFIG;
```

Comment:
Is the client protocol change due to new field coming in a different PR (see: `ClientDynamicClusterConfig#addCardinalityEstimatorConfig`, `AddCardinalityEstimatorConfigMessageTask`)

Reply:
Yes, I ll do the client configs in new PRs. We need this first committed before its visible to the client-protocol anyhow.